wiki/Gameplay/Grade/es.md

```diff
@@ -1,55 +1,64 @@
 ---
-no_native_review: true
-outdated_translation: true
-outdated_since: 299cc666553fbfc6d71cf11ac90a5d88f975d8e6
+tags:
+  - letter grade
+  - rank
+  - silver s
+  - silver ss
+  - sh
+  - xh
+  - rango
+  - s plateada
+  - ss plateada
 ---
 
-# Rangos
+# Grado
+
+*Para otros usos, véase [Rango (desambiguación)](/wiki/Disambiguation/Rank).*
```

Comment:
Tiene sentido esta aclaracion si cambiamos la nomenclatura?

Reply:
I know that in english it does make sense, but for spanish it doesn't really as the names are different now.